src/Common/src/Interop/Windows/mincore/WinRT/Interop.UnsafeCreateFile.cs

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.IO;
+using System.Runtime.InteropServices;
+using Microsoft.Win32.SafeHandles;
+
+internal partial class Interop
+{
+    internal partial class mincore
+    {
+        internal static unsafe SafeFileHandle UnsafeCreateFile(
```

Comment:
I see... related to my previous comment on the other UnsafeCreateFile, it exists because this one exists?  I'm wondering if this one should just be renamed to CreateFile and the other UnsafeCreateFile removed.